haas/model.py

```diff
@@ -49,13 +48,20 @@
 def init_db(uri=None):
     """Start up the DB connection.
 
+    If `create` is True, this will generate the schema for the database, and
+    perform initial population of tables.
+
     `uri` is the uri to use for the databse. If it is None, the uri from the
     config file will be used.
     """
     if uri is None:
         uri = cfg.get('database', 'uri')
     app.config.update(SQLALCHEMY_DATABASE_URI=uri)
 
+# A joining table for networks and projects, which have a many to many relationship:
```

Comment:
Let's pick a name that makes it clear what the relationship here actually is -- projects' *access* to networks.